pennylane/devices/qutrit_mixed/measure.py

```diff
@@ -91,24 +98,35 @@ def calculate_reduced_density_matrix(
 
 
 def calculate_probability(
-    measurementprocess: StateMeasurement, state: TensorLike, is_state_batched: bool = False
+    measurementprocess: StateMeasurement,
+    state: TensorLike,
+    is_state_batched: bool = False,
+    measurement_errors=None,
 ) -> TensorLike:
     """Find the probability of measuring states.
 
     Args:
         measurementprocess (StateMeasurement): measurement to apply to the state.
         state (TensorLike): state to apply the measurement to.
         is_state_batched (bool): whether the state is batched or not.
+        measurement_errors (List[Callable]): List of operators to apply to each wire being measured
+        to simulate measurement error.
 
     Returns:
         TensorLike: the probability of the state being in each measurable state.
     """
     for op in measurementprocess.diagonalizing_gates():
         state = apply_operation(op, state, is_state_batched=is_state_batched)
 
+    wires = measurementprocess.wires
     num_state_wires = get_num_wires(state, is_state_batched)
     wire_order = Wires(range(num_state_wires))
-    wires = measurementprocess.wires
+
+    if measurement_errors is not None:
+        with queuing.QueuingManager.stop_recording():
```

Comment:
This is necessary so that if a qnode with measurment error is ran in another qnode these errors aren't applied to the original qnode.

Reply:
I'm a bit confused by this, when would a QNode be running within another QNode?

Reply:
It's just an edge case, but I one possible case is if you wanted to run a quantum circuit in a quantum function to select a parameter for the next circuit then this could cause an issue. It could silently cause a bug in a case like that so I thought it was best to add this, also a similar implementation occurs in qubit implementation of sampling (occurring in 3 places in `devices.qubit.sampling` lines 118-119, 469-470, and 542-543).